.github/workflows/build-and-test.yaml

```diff
@@ -65,3 +65,33 @@ jobs:
           fi
           cat $FILE
           curl -v -F json_file=@$FILE https://coveralls.io/api/v1/jobs
+
+  build-and-test-fedora:
+    name: fedora
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Setup Podman
+        run: |
+          sudo apt update
+          sudo apt-get -y install podman
+          podman pull fedora:latest
+      - name: Get source
+        uses: actions/checkout@v3
+        with:
+          path: 'GOTCHA'
+      - name: Create container and run tests
+        run: |
+          {
+              echo 'FROM fedora:latest'
+              echo 'RUN dnf -y update'
+              echo 'RUN dnf -y install check-devel cmake git gcc-c++ python3-sphinx sed'
+              echo 'RUN dnf clean all'
+              echo 'COPY GOTCHA GOTCHA'
+              echo 'WORKDIR /GOTCHA'
+              echo 'RUN sed -i "s/-O0 //g" test/hammer/CMakeLists.txt'
```

Comment:
On PR#139 i removed the -O0 flag. So this can be removed now.

Reply:
Done